packages/back-end/src/events/notifiers/EventNotifier.ts

```diff
@@ -34,7 +35,7 @@ export class EventNotifier implements Notifier {
   private static jobHandler(job: Job<EventNotificationData>): void {
     const { eventId } = job.attrs.data;
     webHooksEventHandler(eventId);
-    // slackEventHandler(event);
+    slackEventHandler(eventId);
```

Comment:
Both handlers do this exact thing at the top:

```ts
  const event = await getEvent(eventId);
  if (!event) {
    // We should never get here
    throw new Error("...");
  }
```

We should hoist this up to this `jobHandler` function instead and just pass Event objects to the various handlers.  The way it's currently implemented, there are 2 identical Mongo queries being executed and that will only grow as we add more handler types in the future.

Reply:
That optimization makes sense. I've added that change here 449cf79b8ba11dc78332ad880983b1ce2ae26580